Src/VimCore/VimSettingsInterface.fs

```diff
@@ -62,6 +62,7 @@ module LocalSettingNames =
     let QuoteEscapeName = "quoteescape"
     let EndOfLineName = "endofline"
     let FixEndOfLineName = "fixendofline"
+    let HideMarksName = "hidemarks"

```

Comment:
Should use name `"vsvimhidemarks"` as this is a non-standard setting.

Reply:
I agree, but there are two conventions for the name: `vsvimsetting` (e.g. `vsvimatomicinsert`) and `vsvim_setting` (e.g. `vsvim_useeditordefaults`). I'm thinking the latter is better.